view/frontend/web/js/view/payment/method-renderer/klarnain.js

```diff
@@ -73,6 +75,15 @@ define(
                         ]
                     );
 
+                    this.showFinancialWarning = ko.computed(
```

Comment:
we can add this function on checkoutCommon

Reply:
I am not sure if we can do that, since the state depends on the payment method